website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -30,14 +30,10 @@ To join the private beta, [register your interest](https://docs.google.com/forms
 Before accessing the editor, you should have a dbt Cloud project already set up. This includes a Git repository, data platform connection, environments, and developer credentials. If you don't have this set up, please contact your dbt Cloud Admin.
 
 To access the visual editor:
```

Comment:
sugg either having this all as a bullet in one line or having two bullets. per [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#lists), 

so maybe:

- To access the visual editor, navigate to the left hand side panel.
- Click **Develop** and then select **Visual Editor**

or

To access the visual editor:
- Navigate to the left hand side panel.
- Click **Develop** and then select **Visual Editor**